suggestion.md

```
@@ -0,0 +1,31 @@
+# پیشنهادهای عملی برای GitDM
+
+1) تقویت مالکیت داده در ویوسِت‌ها (اعمال شد)
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)